AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -209,10 +215,8 @@ def fail_fast(message=None):
         raises FailFast
         :return: None
         """
-        Report.info("Failing fast. Raising an exception and shutting down the editor.")
         if message:
             Report.info("Fail fast message: {}".format(message))
-        TestHelper.close_editor()
```

Comment:
This seems like a more general change affecting more than just physics. I know you've already gone to a tonne of extra effort resurrecting this PR, but would it be worth splitting this change into a separate PR?

Reply:
From Alex's comment about that line:

Aaron: Is it safe to remove this? are other tests relying on this?
Alex: No test should ever close the editor. This code is just wrong

I included sig-testing here, they can take a look.

Reply:
Looks sane to me. The only concern is if some test is using this function directly (which it shouldn't) so a main suite pass should be good enough evidence.

Reply:
Thanks @evanchia-ly-sdets, AR for this PR looks green: https://jenkins.build.o3de.org/blue/organizations/jenkins/O3DE/detail/PR-8066/2/pipeline 

Is there anything else I should try to be safe?

Reply:
lgtm